openapi/paths/credential-hashes@avalara.yaml

```
@@ -0,0 +1,70 @@
+get:
+  x-products:
+    - Users
+  parameters:
+    - $ref: ../components/parameters/collectionFilter.yaml
+    - $ref: ../components/parameters/collectionLimit.yaml
+    - $ref: ../components/parameters/collectionOffset.yaml
+    - $ref: ../components/parameters/collectionSort.yaml
+    - $ref: ../components/parameters/collectionQuery.yaml
+  tags:
+    - Avalara credentials
+  summary: Retrieve Avalara credentials
+  operationId: GetAvalaraCredentialHashCollection
+  x-sdk-operation-name: getAllAvalaraCredentials
+  description: |-
+    Retrieves a list of Avalara credentials.
```

```suggestion
  description: Retrieves a list of Avalara credentials.
```
The purpose of the block scalar is multi-line descriptions.